src/pages/organize/[orgId]/journeys/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { GetServerSideProps } from 'next';
+import Head from 'next/head';
+import { useIntl } from 'react-intl';
+import { Box, Grid } from '@material-ui/core';
+
+import JourneyCard from 'components/organize/journeys/JourneyCard';
+import JourneysLayout from '../../../../layout/organize/JourneysLayout';
```

Comment:
@ziggabyte This is another standard thing we should put in the README file - that the `/src` folder is provided as a relative root within any Typescript file. So you can remove all of the `../../../` :-D
You can find the config in `tsconfig.json` -> `"baseUrl": "./src"`

Reply:
ooooooh! fancy cool!

Reply:
@lowlandjuju i changed the imports to remove all the ".../../../" and that makes me not pass the playwright tests, does that mean i did it wrong or i need to change something in the tests..?

Reply:
@ziggabyte You can only use absolute imports in files that are within `src`. In the `playwright` folder you have to use the relative imports (the ones with `../../../` This has to do with how our playwright test runner is configured.

Reply:
Thank you! Everin clarified on slack also :) I updated and now it passes and works!